src/app/pages/TopicPage/Curation/index.jsx

```diff
@@ -5,28 +5,19 @@ import VisuallyHiddenText from '#psammead/psammead-visually-hidden-text/src';
 import CurationGrid from './CurationGrid';
 import HierarchicalGrid from '../HierarchicalGrid';
 import Subheading from './Subhead';
+import { VISUAL_PROMINENCE, VISUAL_STYLE } from '../constants';
 
-export const VISUAL_STYLE = {
-  NONE: 'NONE',
-  BANNER: 'BANNER',
-};
-
-export const VISUAL_PROMINANCE = {
-  NORMAL: 'NORMAL',
-  HIGH: 'HIGH',
-};
-
-// Maps a visual style and prominance to a component that renders that curation
+// Maps a visual style and prominence to a component that renders that curation
 const components = {
   [VISUAL_STYLE.NONE]: {
-    [VISUAL_PROMINANCE.NORMAL]: CurationGrid,
-    [VISUAL_PROMINANCE.HIGH]: HierarchicalGrid,
+    [VISUAL_PROMINENCE.NORMAL]: CurationGrid,
+    [VISUAL_PROMINENCE.HIGH]: HierarchicalGrid,
```

Comment:
Should we consider refactoring this to use our new `getComponent` utility? (Or just leave it 🐉 )

Reply:
There were some dragons, but I thought it was worth doing at this point, so that we don't need to revisit in future